README.md

```diff
@@ -773,6 +810,89 @@ The `WHERE`describes in which part of the app the integer is used. If
 it is used in more than one screen use `all`. The `DESCRIPTION` should
 tell the reader what the actual purpose of the integer is.
 
+<a name="naming--colors"></a><a name="11.6"></a>
+- [11.6](#naming--colors) Colors should be named after the following rules: Do not use prefixes such as dark, light, bright. You cannot ensure that you never need e.g. an even darker blue than your current darkblue in future. Also, do not define a color multiple times because it makes it difficult to apply changes. Instead create a distinct color palette and reference from there. Make sure that you use one of the formats below.
+
+Basic colors like white, black, grey are necessary for fonts, lines, borders, shadows etc. and should be named after the following pattern:
+
+```
+<HUE>_<LIGHTNESS>
+```
+
+```
+// example
+black_80
+black_50
+```
+
+The `HUE` describes the main color. The `LIGHTNESS` part tells how dark or light the color is.
+
+Specific colors depend on the app design/ branding/ corporate identity  and should be named after the following pattern:
+
+```
+<HUE>_<NAME>
+```
+
+```
+// example
+red_pinkish
+green_pastel
+```
+The `HUE`describes the main color. The `NAME` part should be unique and can be generated by tools like [Name that Color](http://chir.ag/projects/name-that-color/#79177E).
+
+Reference colors from this palette to connect them with their usage in the app. However always prefer to use styles and themes for this.
+
+```
+<WHERE>_<DESCRIPTION>
+```
+
+```
+// good
+<color name="purple_honeyflower">#59177e</color>
+<color name="black_100">#000000</color>
+<color name="all_toolbar">@color/purple_honeyflower</color>
+<color name="all_button_fab">@color/purple_honeyflower</color>
+<color name="all_button_primary">@color/purple_honeyflower</color>
+<color name="all_title_primary">black_100</color>
+
+// bad
+<color name="all_toolbar">#59177e</color>
+<color name="all_button_fab">#59177e</color>
+<color name="all_button_primary">#59177e</color>
+<color name="all_title_primary">#000000</color>
+```
+
+The `WHERE` describes  in which part of the app the color is used. The`DESCRIPTION` part should tell the reader what the actual purpose of this color is.
```

Comment:
We discussed this and want to use the name that color plugin. We are not sure if we use name that color or name that material color. Please review this description and adapt accordingly.

EDIT: Some hints on why naming a color based on their purpose is bad design. First of all we often think of a color when we need it and not where it is used. This makes it harder to find the correct one and leads to opening the colors.xml file and hence wastes time. Also it is hard to keep track of them. When there is suddenly a new purpose, which was not foreseen you probably have to rename the color. Therefore you introduce a new variable, which you have to remember. When naming the color by its actual name you would never get to this point and keep a consistent usage over time.